launch/hero2_festival_tts.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
```

Comment:
This is not a hero-specific launch file. If we want to keep this it should be moved to robot_launch_files

Reply:
Agreed